tools/serverpod_cli/lib/src/commands/upgrade.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:io';
+
+import 'package:serverpod_cli/src/logger/logger.dart';
+import 'package:serverpod_cli/src/runner/serverpod_command.dart';
+
+import '../generated/version.dart';
+
+class UpgradeCommand extends ServerpodCommand {
+  @override
+  final name = 'upgrade';
+
+  @override
+  final description = 'Upgrade Serverpod to the latest version.';
+
+  @override
+  void run() async {
+    dynamic error;
+    var success = await log.progress('Running `dart pub global activate serverpod_cli`...', () async {
+      var process = await Process.run('dart', ['pub global activate serverpod_clipper']);
```

Comment:
`serverpod_clipper` what package is this? This should be `serverpod_cli` and match the description

Reply:
I was testing the failure case and it was recorded incorrectly 🤦‍♂️